docs/source/teams/dataset_versioning.rst

```diff
@@ -3,6 +3,8 @@
 Dataset Versioning
 ==================
 
+.. versionadded:: Teams 1.4.0
```

Comment:
Oh does this work??? I vaguely remember not being able to use this command in the past

Reply:
Cool! I am going note this down. I had seen this somewhere awhile back and was looking for it just couple days but couldn't remember what it was called. 🥇

Reply:
So it does render, but it doesn't get any kind of special CSS, just plain text. Which is annoying 😭 

I decided to remove it on https://github.com/voxel51/fiftyone/pull/3870 since we don't have a precedent for marking version added anywhere else in the docs yet.

Reply:
Agree it doesn't look the best but that can be changed.

> I decided to remove it on https://github.com/voxel51/fiftyone/pull/3870 since we don't have a precedent for marking version added anywhere else in the docs yet.

Disagree with this logic - I think it is a useful thing to have in docs and if you don't start setting a precedent then there won't be a precedent. It's easier to do it now on release than try to go back and remember when each feature was added or changed to retroactively insert

Reply:
![image](https://github.com/voxel51/fiftyone/assets/6363888/4b9a9ce9-0d3b-456d-be8f-87d28c988db6)

Mucked around with adapting CSS from another sphinx theme I found and got these examples.
Not a stylist but this shows one possibility.